webrtc.html

```diff
@@ -5945,10 +5945,24 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       <div>
         <pre class="idl">dictionary RTCRtpCodecCapability {
              DOMString mimeType;
+             unsigned long  clockRate;
+             unsigned short channels = 1;
```

Comment:
I put in a default value since setCodecPreferences takes a sequence of RTCRtpCodecCapability as an argument.